canvas_modules/common-canvas/src/icons/icon.jsx

```diff
@@ -90,6 +91,12 @@ export default class Icon extends React.Component {
 			return <ChevronLeft16 className={className} disabled={this.props.disabled} />;
 		case (CARBON_ICONS.CHEVRONARROWS.RIGHT):
 			return <ChevronRight16 className={className} disabled={this.props.disabled} />;
+		case (CARBON_ICONS.ARROWS.UP):
```

Comment:
I would move these to where they are used.  We should stop adding new icons since this causes all icons to be loaded in our every component.